warehouse/models/intermediate/payments/int_payments__refunds_to_aggregations.sql

```diff
@@ -0,0 +1,58 @@
+{{ config(materialized = 'table',) }}
+
+WITH refunds AS (
+    SELECT *
+    FROM {{ ref('int_payments__refunds_deduped') }}
+),
+
+summarize_by_type AS (
+    SELECT
+        aggregation_id,
+        retrieval_reference_number,
+        approval_status,
+        SUM(refund_amount) AS refund_amount,
```

Comment:
I think this needs to be `proposed_amount` (this is what I've been complaining about, their column naming is incredibly counterintuitive -- `proposed_amount` is `The actual amount refunded to the customer.` while `refund_amount` is `The amount equalling transaction_amount - proposed_amount.`)

Reply:
ah jeeze okay then I need to change how I prepare the micropayments table for the union in `int_payments__refunds_rdeduped`, doing that now

Reply:
Ope wow yeah sorry I missed it above!